src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -7,25 +7,46 @@ import snyk.amplitude.api.AmplitudeExperimentApiClient
 import snyk.amplitude.api.AmplitudeExperimentApiClient.Defaults.FALLBACK_VARIANT
 import snyk.amplitude.api.ExperimentUser
 import snyk.amplitude.api.Variant
+import java.io.IOException
+import java.util.Properties
 import java.util.concurrent.ConcurrentHashMap
 
+private val LOG = logger<AmplitudeExperimentService>()
+
 @Service
 class AmplitudeExperimentService : Disposable {
-    private val log = logger<AmplitudeExperimentService>()
 
-    private val apiClient: AmplitudeExperimentApiClient =
-        AmplitudeExperimentApiClient.create(apiKey = "client-bQAwhmzLhgpgqAhKAqRzIjnS915YrAqj")
+    private var amplitudeLoaded = false
+    private var apiClient: AmplitudeExperimentApiClient = AmplitudeExperimentApiClient.create(apiKey = "")
```

Comment:
`lateinit var` to avoid fake initialization

Reply:
I like the idea, will add it in separate commit 😉